src/install/lsc-core.spec

```diff
@@ -103,121 +94,130 @@ install -p -m 0644 lib/systemd/system/lsc-async@.service %{buildroot}%{_unitdir}
-sed -i 's:^LOG_DIR.*:LOG_DIR="%{lsc_logdir}":' %{buildroot}/usr/bin/lsc %{buildroot}/usr/bin/lsc-agent %{buildroot}/usr/bin/hsqldb
-sed -i 's:^VAR_DIR.*:VAR_DIR="/var/lsc":' %{buildroot}/usr/bin/hsqldb
+sed -i \
+  -e 's:^CFG_DIR.*:CFG_DIR="%{_sysconfdir}/lsc":' \
+  -e 's:^LIB_DIR.*:LIB_DIR="%{_libdir}/lsc":' \
+  -e 's:^LOG_DIR.*:LOG_DIR="%{lsc_logdir}":' \
+  %{buildroot}%{_bindir}/lsc \
+  %{buildroot}%{_bindir}/lsc-agent \
+  %{buildroot}%{_bindir}/hsqldb
+sed -i \
+  -e 's:^VAR_DIR.*:VAR_DIR="%{_sharedstatedir}/lsc":' \
+  -e 's:^HSQLDB_PIDFILE.*:HSQLDB_PIDFILE="%{_rundir}/hsqldb.pid":' \
+  %{buildroot}%{_bindir}/hsqldb
+
+
+%pre
+getent group lsc > /dev/null 2>&1 || groupadd --system lsc
+getent passwd lsc > /dev/null 2>&1 || \
+  useradd --system --gid lsc \
+   --home-dir %{_sharedstatedir}/lsc \
+   --shell "/sbin/nologin" \
+   --comment "LDAP Synchronization Connector user" \
+   lsc
 
 %post
 %systemd_post lsc-async.service
 %systemd_post lsc-async@.service
 %systemd_post lsc-sync.service
 %systemd_post lsc-sync@.service
 
-# Always do this
-# Create user and group if needed
-getent group %{lsc_group} > /dev/null 2>&1 || groupadd --system %{lsc_group}
-getent passwd %{lsc_user} > /dev/null 2>&1 || useradd --system --gid %{lsc_group} --home-dir /etc/lsc %{lsc_user}
-# Change owner
-/bin/chown -R %{lsc_user}:%{lsc_group} %{lsc_logdir}
-
-# Add symlink for sample to work
-ln -sf /usr/%{_lib}/lsc/ /usr/share/doc/lsc/%{_lib}
-ln -sf /usr/bin/lsc /usr/share/doc/lsc/bin/
-
 %preun
 %systemd_preun lsc-async.service
 %systemd_preun lsc-sync.service
 
-%postun
-#=================================================
-# Post uninstallation
-#=================================================
-
-# Don't do this if newer version is installed
-if [ $1 -eq 0 ]
-then
-	# Remove sample symlinks
-	rm -rf /usr/share/doc/lsc/%{_lib}
-	rm -rf /usr/share/doc/lsc/bin/
-
-        # Delete user and group
-        /usr/sbin/userdel -r %{lsc_user}
-fi
-
-#=================================================
-# Cleaning
-#=================================================
-%clean
-rm -rf %{buildroot}
 
-#=================================================
-# Files
-#=================================================
 %files
-%defattr(-, root, root, 0755)
-%config(noreplace) /etc/lsc/
-%config(noreplace) /etc/cron.d/lsc
-%config(noreplace) /etc/default/lsc-sync
-%config(noreplace) /etc/default/lsc-async
-/usr/bin/lsc
-/usr/bin/lsc-agent
-/usr/bin/hsqldb
+%license LICENSE.txt
+%doc README.md doc/html/
+%doc sample/ etc/lsc.xml-sample etc/sql-map-config.xml-sample
+%doc etc/sql-map-config.d/InetOrgPerson.xml-sample
+%dir %{_sysconfdir}/lsc/
+%config(noreplace) %{_sysconfdir}/lsc/*.xml
+%dir %{_sysconfdir}/lsc/sql-map-config.d/
+%config(noreplace) %{_sysconfdir}/lsc/sql-map-config.d/InetOrgPerson.xml
+%config(noreplace) %{_sysconfdir}/cron.d/lsc
+%config(noreplace) %{_sysconfdir}/default/lsc
```

Comment:
I think some lines have been dropped during the rebase.

Here we should:

1. remove: (this file does not exist anymore)

```
%config(noreplace) %{_sysconfdir}/default/lsc
```

2. add:

```
%config(noreplace) %{_sysconfdir}/default/lsc-sync
%config(noreplace) %{_sysconfdir}/default/lsc-async
```

All other modification seems, ok. Thanks for you hard work!

Reply:
good catch, fixed.